tests/test_totals.py

```diff
@@ -34,7 +34,11 @@
     'last_debts_owed_by_committee': 42,
     'net_contributions': 123,
     'net_operating_expenditures': 321,
-    'last_debts_owed_to_committee': 42
+    'last_debts_owed_to_committee': 42,
+    'cmte_nm': None,
```

Comment:
Using a string will create better tests since we expect their to be a string for these

Reply:
done.